packages/web-frontend/src/actions.js

```diff
@@ -554,6 +556,18 @@ export function updateMeasureConfig(measureId, measureConfig) {
   };
 }
 
+/**
+ * Updates measure config for current measure in measureBar once the hierarchy is populated
+ *
+ * @param {object} measureConfig
+ */
+export function updateCurrentMeasureConfigOnceHierarchyLoads(periodString) {
```

Comment:
```suggestion
export function updateMeasureDateRangeOnceHierarchyLoads(periodString) {
```

(can also keep `Current` in the name if you thing it's required)